pennylane/interfaces/batch/jax_jit.py

```diff
@@ -0,0 +1,298 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.

```

Comment:
@antalszava were any changes made to this file after renaming it?

Reply:
Yes, now I've changed to using `qml.math.unstack` there too.